### ISSUE TITLE
Streamlined Tagging Experience

### ISSUE THREAD

**Issue by fritzdavenport** [lucyparsons/OpenOversight#775]
As a photo tagger, it would be a nice experience to the information required to tag an officer all contained in the view that I'm using. I'd like a form that looks up name / badge and shows the officer details inline on the page.

**Comment by fritzdavenport** [comment]
![Screenshot from 2020-07-27 23-08-54](https://user-images.githubusercontent.com/5983782/88615081-2d60a380-d05f-11ea-8626-1ace930248f5.png)

a hacked together version of what I'd imagine

**Comment by brianmwaters** [comment]
As someone who is going to be tagging a very large number of photos in the coming days, this would probably be nice, but I haven't felt that pain yet ;)

**Comment by abandoned-prototype** [comment]
Yes. I 100% agree that this is an important thing to add. Especially, I would like the lookup to show the best match(es) and not just fail if there is no match because a single letter is off. Sometimes the badge number and name is hard to read, but all together they still allow for a match with pretty good confidence.
